python/arcticdb/version_store/_normalization.py

```diff
@@ -842,7 +860,13 @@ def normalize(self, obj, **kwargs):
         column_val = np.array(memoryview(self._buffer[: size * 8]), np.uint8).view(np.uint64)
 
         return NormalizedInput(
-            item=NPDDataFrame(index_names=[], index_values=[], column_names=["bytes"], columns_values=[column_val]),
+            item=NPDDataFrame(
+                index_names=[],
+                index_values=[],
+                column_names=["bytes"],
+                columns_values=[column_val],
+                sorted=_SortedValue.UNKNOWN,
```

Comment:
I think this should be `UNSORTED` for pickled data?

Reply:
`UNKNOWN` is probably better considering the deserialized data could have a sorted index.

Reply:
I will leave as is now, since assuming unsorted could cause unnecessary exceptions, in case we deal with the scenario that Qi commented

Reply:
We can't append or update pickled data, so if that's all this enum is for I still think it should be `UNSORTED`, but not a hill I'm willing to die on.